site/en/docs/extensions/reference/storage/index.md

```diff
@@ -24,13 +24,17 @@ Even though extensions can use the [`Storage`][mdn-storage] interface (accessibl
 - Content scripts share storage with the host page.
 - Data saved using the `Storage` interface is lost when the user clears their browsing history.
 
+To move data from web storage APIs to extension storage APIs from a service worker:
```

Comment:
This should be in the migration guide. (This is the same as the other one, isn't it?)

Reply:
Hmm, let me check on these. I need to figure out what I intended, as you mention it seems like something went wrong 😅

Reply:
Ah, so if you look at the changed lines the only diff here is adding the first line to clarify what we are describing. The migration steps are already here before this PR so I am intentionally not touching them so you can move in yours. Hopefully that makes sense.

Reply:
Here's the PR for moving this: https://github.com/GoogleChrome/developer.chrome.com/pull/7174